standalone-metastore/src/main/thrift/hive_metastore.thrift

```diff
@@ -893,9 +884,11 @@ struct GetValidWriteIdsResponse {
 
 // Request msg to allocate table write ids for the given list of txns
 struct AllocateTableWriteIdsRequest {
-    1: required list<i64> txnIds,
+    1: optional list<i64> txnIds,
     2: required string dbName,
     3: required string tableName,
+    4: optional string replPolicy,
+    5: optional list<TxnToWriteId> txnToWriteIdList,
```

Comment:
rename as srcTxnToWriteIdList. Also, add a comment that txn and write ids in this list are assumed to be sorted in ascending order and are in continuous sequence.

Reply:
done